src/conv/k-g/k-g.cpp

```diff
@@ -162,7 +165,72 @@ int main
 					point2.outerDiameter = beamSection.TS2;
 					point2.innerDiameter = beamSection.TT2;
 
-					std::string beamNumber = std::to_string(*itr);
+					geometry.addPipePnt(point1);
+					geometry.addPipePnt(point2);
+				    }
+				    else if (beamSection.CST == 2) {
+
+				    }
+				    else if ((beamSection.sectionType != "") && (beamSection.sectionType != "SECTION_08") && (beamSection.sectionType != "SECTION_09")) {
+					point_t point1;
+					point_t point2;
+					point_t point3;
+
+					point1[X] = kData.nodes[n1].x * factor;
+					point1[Y] = kData.nodes[n1].y * factor;
+					point1[Z] = kData.nodes[n1].z * factor;
+
+					point2[X] = kData.nodes[n2].x * factor;
+					point2[Y] = kData.nodes[n2].y * factor;
+					point2[Z] = kData.nodes[n2].z * factor;
+
+					point3[X] = kData.nodes[n3].x * factor;
+					point3[Y] = kData.nodes[n3].y * factor;
+					point3[Z] = kData.nodes[n3].z * factor;
+
+					int D1 = beamSection.D[0];
+					int D2 = beamSection.D[1];
+					int D3 = beamSection.D[2];
+					int D4 = beamSection.D[3];
+

```

Comment:
I get the following compilation errors here:
```
[ 79%] Building CXX object src/conv/k-g/CMakeFiles/k-g.dir/k-g.cpp.o
/home/rossberg/Devel/BRL-CAD/brlcad.devel_k-g/src/conv/k-g/k-g.cpp: In function ‘int main(int, char**)’:
/home/rossberg/Devel/BRL-CAD/brlcad.devel_k-g/src/conv/k-g/k-g.cpp:191:45: error: unused variable ‘D1’ [-Werror=unused-variable]
  191 |                                         int D1 = beamSection.D[0];
      |                                             ^~
/home/rossberg/Devel/BRL-CAD/brlcad.devel_k-g/src/conv/k-g/k-g.cpp:192:45: error: unused variable ‘D2’ [-Werror=unused-variable]
  192 |                                         int D2 = beamSection.D[1];
      |                                             ^~
/home/rossberg/Devel/BRL-CAD/brlcad.devel_k-g/src/conv/k-g/k-g.cpp:193:45: error: unused variable ‘D3’ [-Werror=unused-variable]
  193 |                                         int D3 = beamSection.D[2];
      |                                             ^~
/home/rossberg/Devel/BRL-CAD/brlcad.devel_k-g/src/conv/k-g/k-g.cpp:194:45: error: unused variable ‘D4’ [-Werror=unused-variable]
  194 |                                         int D4 = beamSection.D[3];
      |                                             ^~
cc1plus: all warnings being treated as errors
```

Reply:
yes, we don't need those, I will delete them